### DISCUSSION TITLE
How to handle free shares?

### DISCUSSION THREAD

**Discussion by RealCommerzpunk** [ghostfolio/ghostfolio#3847] [Q&A]
Hello,
I am lucky, my employer offers me free shares from time to time. To get them, I have to buy an amount and get another amount for free.

How do I add these to my Portfolio in Ghostfolio the best way? If I consider them as "buy" with a price 0, I get huge drops in my chart, because GF of course thinks, they had a value of 0 and the next day they shoot to market price.

I don't add them with the price they had the day I got them, because I want to keep track of the fact that they are free for me. So I like to see the impact on my performance percentage, because I made a huge profit in percentage, but I dont like to see the drop is the shares value itself.

And ideas? 

![image](https://github.com/user-attachments/assets/d87e2eaa-c7cb-4eb4-89f0-2f9494da6261)

**Comment by gstr3745** [comment]
As in a previous discussion I cannot find right now, I assume the right way to handle this is to let the market price value "override" the purchase value. 
But I don't have the ability to edit the code and to propose this :(
